sdk/hatchery/hatchery.go

```diff
@@ -68,7 +67,7 @@ type CommonConfiguration struct {
 // ID returns hatchery id
 type Interface interface {
 	Init() error
-	SpawnWorker(model *sdk.Model, jobID int64, requirements []sdk.Requirement, registerOnly bool, logInfo string) (string, error)
+	SpawnWorker(model *sdk.Model, isWorkflowJob bool, jobID int64, requirements []sdk.Requirement, registerOnly bool, logInfo string) (string, error)
```

Comment:
humm don`'t you think there is too much arguments in this function ?
passing all spawn options through a struct could be nice

Reply:
done